CK3toEU4/Source/EU4World/Country/Country.cpp

```diff
@@ -999,72 +1003,132 @@ void EU4::Country::assignReforms(const std::shared_ptr<mappers::RegionMapper>& r
 	// RELIGIONS
 	// Muslims
 	std::set<std::string> muslimReligions = {"sunni", "zikri", "yazidi", "ibadi", "kharijite", "shiite", "druze", "hurufi", "qarmatian"};
-	// Mazdans - TODO(#41): Re-enable when we can map to these religions
-	// std::set<std::string> mazdanReligions = { "zoroastrian", "zurvanism", "gayomarthianism", "mazdaki", "manichean", "khurmazta", "khurramism", "urartuism" };
+	for (const auto& religion: religionMapper.getGeneratedReligions())
+		if (religion.religionGroup == "muslim")
+			muslimReligions.insert(religion.name);
+	// Mazdans
+	std::set<std::string> mazdanReligions = {"zoroastrian", "zurvanism", "gayomarthianism", "mazdaki", "manichean", "khurmazta", "khurramism", "urartuism"};
+	for (const auto& religion: religionMapper.getGeneratedReligions())
+		if (religion.religionGroup == "zoroastrian_group")
+			mazdanReligions.insert(religion.name);
 	// Buddhists - TODO(#41): Re-enable when we can map to these religions
 	// std::set<std::string> buddhistReligions = { "buddhism", "vajrayana", "mahayana" };
 	// Eastern
 	std::set<std::string> easternReligions = {"confucianism", "shinto", "buddhism", "vajrayana", "mahayana"};
+	for (const auto& religion: religionMapper.getGeneratedReligions())
+		if (religion.religionGroup == "eastern")
+			easternReligions.insert(religion.name);
 	// Indians (Dharmic + Buddhists)
 	std::set<std::string> indianReligions = {"buddhism", "vajrayana", "mahayana", "hinduism", "jain"};
+	for (const auto& religion: religionMapper.getGeneratedReligions())
+		if (religion.religionGroup == "dharmic" || religion.parent == "buddhism" || religion.parent == "vajrayana" || religion.parent == "mahayana")
+			indianReligions.insert(religion.name);
 	// Protestants
 	std::set<std::string> protestantReligions = {"protestant", "reformed", "hussite", "cathar", "waldensian", "lollard", "adamites"};
+	for (const auto& religion: religionMapper.getGeneratedReligions())
+		if (religion.religionGroup == "christian" && (religion.country.contains("uses_church_power") || religion.country.contains("fervor") ||
+																		 religion.country.contains("uses_anglican_power") || religion.country.contains("uses_hussite_power")))
```

Comment:
While lovely, .contains() on a string is still unsupported  on linux. :/
Replace with regular `.find("uses_church_power") != std::string::npos` etc.